tests/unit/modules/test_virt.py

```diff
@@ -1850,6 +1850,28 @@ def test_get_xml(self):
         self.assertEqual(xml, virt.get_xml("test-vm"))
         self.assertEqual(xml, virt.get_xml(domain))
 
+    def test_cpu_baseline(self):
+        """
+        Test virt.cpu_baseline()
+        """
+        capabilities_xml = """<capabilities>
```

Comment:
@rst0git Please use `tests.support.helpers.dedent` on this block.

Reply:
Thank you for the code review! I have updated the pull request.